group.yml

```diff
@@ -26,76 +28,123 @@ sources:
   rhel-7-server-rhceph-3-tools-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhceph-tools/3/os/
+      baseurl:
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/rhceph-tools/3/os/
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/rhceph-tools/3/os/
     content_set:
       default: rhel-7-server-rhceph-3-tools-rpms
+      optional: true
+      ppc64le: rhel-7-server-rhceph-3-for-power-le-tools-rpms
   rhel-fast-datapath-htb-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/htb/rhel/server/7/x86_64/fast-datapath/os/
+      baseurl:
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/beta/rhel/power-le/7/ppc64le/fast-datapath/os/
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/htb/rhel/server/7/x86_64/fast-datapath/os/
     content_set:
       default: rhel-7-fast-datapath-htb-rpms
+      optional: true
+      ppc64le: rhel-7-for-power-le-fast-datapath-beta-rpms
   rhel-fast-datapath-rpms:
     conf:
       baseurl:
-        signed: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/fast-datapath/os/
-        unsigned: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/fast-datapath/os/
+        unsigned:
+          ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/fast-datapath/os/
+          x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/fast-datapath/os/
+        signed:
+          ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/fast-datapath/os/
+          x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/fast-datapath/os/
     content_set:
       default: rhel-7-fast-datapath-rpms
+      ppc64le: rhel-7-for-power-le-fast-datapath-rpms
   rhel-server-extras-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/extras/os/
+      baseurl:
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/extras/os/
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/extras/os/
     content_set:
       default: rhel-7-server-extras-rpms
+      ppc64le: rhel-7-for-power-le-extras-rpms
   rhel-server-optional-rpms:
     conf:
-      baseurl: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/optional/os/
+      baseurl:
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/optional/os/
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/optional/os/
       enabled: 1
```

Comment:
curious why we would need to specify "enabled". isn't that the default? when would we ever add a repo on an image and not want it enabled? (except for testing purposes i suppose)

Reply:
`enabled: 1` in group.yml makes it enabled on ALL images by default. That probably shouldn't be there but it has been forever and removing it would prob break something.

Reply:
so we enable the optional repo for all images? ugh. that seems likely to end in tragedy at some point. now that we can easily run local builds to test we (I) oughta block some time and see what actually goes wrong if we remove this... i bet nothing.